lib/commander/runner.rb

```diff
@@ -329,13 +329,9 @@ def require_valid_command(command = active_command)
 
     def remove_global_options(options, args)
       options.each do |option|
-        switches = option[:switches].dup
+        switches = option[:switches]
         next if switches.empty?
-
-        if (switch_has_arg = switches.any? { |s| s =~ /[ =]/ })
-          switches.map! { |s| s[0, s.index('=') || s.index(' ') || s.length] }
-        end
-
+        switch_has_arg = switches.any? { |s| s =~ /[ =]/ }
```

Comment:
I was wondering if removing the `map!` would have any effect on `expand_optionally_negative_switches` - however it looks like that method is already using a regexp and would not care about the remainder.